demo/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mhart/alpine-node:6.8
+
+COPY . /var/www/paale-dai
+
+WORKDIR /var/www/paale-dai
+
+RUN apk add --no-cache curl tar && curl -o- -L https://yarnpkg.com/install.sh | sh
+ENV PATH /root/.yarn/bin:$PATH
+
+RUN chown -R root:root /var/www/paale-dai\
```

Comment:
Root:root is not ideal, either create a user or use nobody: nobody depending on Linux flavor.